package.json

```diff
@@ -1,42 +1,66 @@
 {
-  "name": "@mediamonks/eslint-config-root",
-  "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
+  "name": "@mediamonks/eslint-config",
+  "version": "2.6.0",
+  "publishConfig": {
+    "access": "public"
+  },
   "license": "MIT",
   "author": "frontend.monks",
+  "homepage": "https://github.com/mediamonks/eslint-config#readme",
+  "bugs": {
+    "url": "https://github.com/mediamonks/eslint-config/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mediamonks/eslint-config.git"
+  },
+  "description": "Sharable eslint config based on Media.Monks Frontend Coding Standards",
+  "keywords": [
+    "Media.Monks",
+    "eslint",
+    "eslint-config"
+  ],
+  "type": "module",
+  "exports": {
+    ".": "./src/index.js"
+  },
```

Comment:
Should we also export the non-index files, which might make it easier to construct your own presets from these building blocks?

Reply:
I don't mind but it also encourages creating custom presets. We can just re-export the rules from the index.js file

Reply:
That also works for me :)